pom.xml

```
@@ -97,7 +97,7 @@
         <pmd.plugin.version>3.13.0</pmd.plugin.version>
         <ant.version>1.10.8</ant.version>
         <javadoc.plugin.version>3.2.0</javadoc.plugin.version>
-        <antlr.version>4.7</antlr.version>
+        <antlr.version>4.7.1</antlr.version>
```

```suggestion
        <antlr.version>4.7.2</antlr.version>
```

The latest 4.7.x version is 4.7.2 -> https://search.maven.org/artifact/org.antlr/antlr4-runtime